ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -13,8 +30,11 @@ tag_specification:
   namespace: ocp
 tests:
 - artifact_dir: /tmp/artifacts
-  as: unit
-  commands: echo "stub"
+  as: registry-unit
+  commands: |
+    pushd staging/operator-registry
+    make unit
+    popd
```

Comment:
At least currently, running something like `make -f staging/operator-registry/Makefile unit` is problematic as the unit target is hardcoded to reference the `./pkg/...` packages, and changing that to use an absolute path is still problematic due to how the submodules are currently defined. Not sure if there's something more robust we could be doing besides changing the directory to that staging repository, running tests, and then jumping back to the root directory.

Reply:
I guess another alternative would be having the root Makefile drive this, but that likely leads to situations where the root Makefile and the corresponding staging Makefile change but that likely doesn't matter much in the context of how we would run units tests in the registry repository.